data/proxy/conf/local.toml

```diff
@@ -1,5 +1,10 @@
 debug = true
 
+[tezos]
+whitelistedRolling = [
```

Comment:
To be consistent, could be rename whitelisted to whitelistedArchive ? And same for the other properties

Reply:
Disagree on that. The goal is to redirect the request  .... by default we redirect to archive nodes. whitelistedRolling represents patterns of url for which will redirect to rolling nodes. Maybe we can rename whitelistedRolling into patternsRolling ?